### PR TITLE
[Improvement][Service] Modify ordinary users tenants

### PR DESCRIPTION
**Pull request by zombietramp**
1: Modify ordinary users and disable the function of selecting other tenants       Scenario: After the bank's internal administrator creates a new ordinary user and assigns a tenant. No other tenants can be selected after the new process

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

### CODE REVIEW

**Comment by chengshiwen** [comment]
@zombietramp Please use English, and resolve the conflicts.
By the way, the v-show problem on template has been solved in pr #5145

**Comment by CalvinKirs** [comment]
Thanks for your contribution. Your PR title does not seem to match the description.
Regarding multi-line SQL execution, can you describe your specific implementation?

File: dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.xml [hunk 1]
```diff
@@ -149,8 +150,7 @@
         from t_ds_process_instance t
         join t_ds_process_definition d on d.id=t.process_definition_id
         join t_ds_project p on p.id=d.project_id
-        where 1 = 1
-        and t.is_sub_process = 0
+        where  and t.is_sub_process = 0
```
Comment (ruanwenjun) [inline dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapper.xml]: Should remove `and`

File: dolphinscheduler-common/sql/soft_version [hunk 2]
```diff
@@ -0,0 +1 @@
+32432423
```
Comment (ruanwenjun) [inline dolphinscheduler-common/sql/soft_version:1]: ignore this file

File: dolphinscheduler-common/test/testFile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+正正正faffdasfasdfas
```
Comment (ruanwenjun) [inline dolphinscheduler-common/test/testFile.txt:1]: ignore this file

File: dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/UserMapper.xml [hunk 4]
```diff
@@ -19,7 +19,7 @@
 <!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd" >
 <mapper namespace="org.apache.dolphinscheduler.dao.mapper.UserMapper">
     <sql id="baseSql">
-       id, user_name, user_password, user_type, email, phone, tenant_id, create_time, update_time, queue, state
+       id, user_name, user_password, user_type, email, phone, tenant_id, create_time, update_time, queue, `state`
```
Comment (ruanwenjun) [inline dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/UserMapper.xml]: For the Sql style consistency, maybe it better not to make this modification ~~

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java [hunk 5]
```diff
@@ -42,17 +42,17 @@
...
 import java.sql.ResultSet;
 import java.sql.ResultSetMetaData;
 import java.sql.SQLException;
 import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
+import java.util.*;
```
Comment (ruanwenjun) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java]: The use of import * is considered a bad practice

**Comment by songjianet** [comment]
Thank you for your contribution, but since this issue has not been dealt with for a long time, I will close it. If you still have problems in the future, you can re-submit a new issue.

**Pull request closed without merging**